[14.0][ADD] stock_restrict_by_planned_consumed_date (#1509)
OCA/stock-logistics-workflow: stock_restrict_by_planned_consumed_date/readme/DESCRIPTION.rst

> This module avoid to reserve goods if the planned consumed date by
> the customer is after expiration time.

Reviewer: ```suggestion
This module avoids to reserve goods having an expiration time prior to the customer's planned consumed date.
```
